src/index.ts

```diff
@@ -48,16 +48,16 @@ const buildVercel = async () => {
   console.log("⚡️ Building project with 'npx vercel build'...");
   console.log("⚡️");
 
-  const vercelBuild = spawn("npx", ["vercel", "build"]);
+  const vercelBuild = exec("npx vercel build");
```

Comment:
I __think__ it would be best to stick with `spawn` if possible, what about doing something like this instead?
```suggestion
  const npx = process.platform === "win32" ? "npx.cmd" : "npx";
  const vercelBuild = spawn(npx, ["vercel", "build"]);
```
instead?
(based on this [stackoverflow thread](https://stackoverflow.com/questions/43230346/error-spawn-npm-enoent))